include/bluetooth/mesh/models.h

```diff
@@ -0,0 +1,74 @@
+/** @file
+ *  @brief Bluetooth Mesh Models API.
+ */
+
+/*
+ * Copyright (c) 2018 Intel Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#ifndef ZEPHYR_INCLUDE_BLUETOOTH_MESH_MODELS_H_
+#define ZEPHYR_INCLUDE_BLUETOOTH_MESH_MODELS_H_
+
+/* GENERIC BATTERY LEVEL (0x100C) */
+
+/* Generic Battery Level defines */
+#define MODEL_BATTERY_SRV_CHARGE_LEVEL_UNKNOWN 0xFF
```

Comment:
Everything in include/bluetooth must be prefixed with BT_. Everything in include/bluetooth/mesh must be prefixed with BT_MESH_.

Reply:
Ok, I'll update it